### DISCUSSION TITLE
[react-relay] is outdated

### DISCUSSION THREAD

**Discussion by fdutey** [DefinitelyTyped/DefinitelyTyped#55971] [Issues with a @types package]
types for `react-relay` package are stuck at version 11.0.2 when relay has moved to 12.0.0 and every other package has been updated.

Leads to types incompatibility between react-relay and relay-runtime (AFAIK) because `Environment` type has actually changed.
Needs a lot of casting to "transfer" types from one library to the other (`as unknown as ReactRelayEnvironment`) and `// FIXME:` comments to keep track

[a comment by an automated account is not shown]

**Reply by maraisr** [reply]
Ah thanks for this one! I'll action this. There is actually quite a number of line-ups that need to happen for v12. 

I'll action this—this week and get a release out.

**Reply by fdutey** [reply]
Oh, awesome, I'll delay my relay update (it's also breaking with material-ui in `vite build` for some reason, don't ask) then. 

Thank you a lot
